src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const element = document.getElementsByClassName('population');
+let allPopulationNumber = 0;
+
+for (const key of element) {
+  allPopulationNumber += +key.innerHTML.replaceAll(',', '');
+}
+
+const averagePopulation = allPopulationNumber / element.length;
+
+const formattedPopulation = averagePopulation.toLocaleString('uk-UA');
+
+const classTotalPopulation = document.querySelector('.total-population');
+
+const formattedallPopulationNumber = averagePopulation.toLocaleString('uk-UA');
```

Comment:
The variable `formattedallPopulationNumber` is incorrectly assigned the formatted average population instead of the total population. It should format `allPopulationNumber` instead of `averagePopulation`.